pkg/apis/serving/v1beta1/inference_service_status.go

```diff
@@ -42,6 +43,8 @@ type InferenceServiceStatus struct {
 	URL *apis.URL `json:"url,omitempty"`
 	// Statuses for the components of the InferenceService
 	Components map[ComponentType]ComponentStatusSpec `json:"components,omitempty"`
+	// Model related statuses
+	ModelStatus ModelStatus `json:"modelStatus,omitempty"`
```

Comment:
General thought regarding potential terminology confusion...

In modelmesh "model" generally refers a concrete/immutable model behind a particular predictor, kind of equivalent to a revision in knative. That's why we have individual model states within the struct, but at this level it should really be called `predictorStatus` I think? Thoughts?

Reply:
As discussed in the community meeting, we may want to differentiate model status and predictor service status as when a new model load fails the predictor service status can still be ready serving the previous model version.